src/Evolution/DgSubcell/SliceVariable.hpp

```diff
@@ -0,0 +1,69 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <algorithm>
+#include <cstddef>
+
+#include "DataStructures/Index.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Domain/Structure/Direction.hpp"
+#include "Domain/Structure/DirectionMap.hpp"
+#include "Evolution/DgSubcell/SliceData.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace evolution::dg::subcell {
+/// @{
+/*!
+ * \brief Slice a Variables object on subcell mesh for a given direction and
+ * slicing depth (number of ghost points).
+ *
+ * This is essentially a wrapper of
+ * `evolution::dg::subcell::detail::slice_data_impl` function, but note that the
+ * last argument has the type `Direction<Dim>`, not a DirectionMap.
+ */
+template <size_t Dim, typename TagList>
+void slice_variable_for_subcell(
```

Comment:
why add `for_subcell`, isn't that redundant with being in the `subcell` namespace? It's not clear what you are trying to differentiate here